### DISCUSSION TITLE
Requesting `Sheet.FlatList` and `Sheet.SectionList`

### DISCUSSION THREAD

**Discussion by likeSo** [tamagui/tamagui#1925] [Ideas]
For now, `FlatList` and `SectionList` will break `Sheet`'s pan down gesture.
Need these two components work like `Sheet.ScrollView`

**Comment by longnguyen2508** [comment]
Got the same issue. Could not swipe down FlatList

**Comment by eruveo** [comment]
I found a workaround to implement FlashList with Sheet, but there are no props to adjust the drag direction behaviour. I can either have a section list or a bottom sheet, as the scroll is then unusable, which is extremely limiting. Would appreciate a quick fix and more props to work with.
